src/utils/API/index.js

```diff
@@ -112,6 +132,28 @@ export const getApplication = (email, onSuccess, onFailure) => {
     .catch((error) => onFailure(error));
 };
 
+// Return an individual application. ID should be the application id.
+export const getApplicationById = (applicationId, onSuccess, onFailure) => {
+  var requestOptions = {
+      method: "GET",
+      headers: {
+        "Access-Control-Allow-Origin": "*",
+        "Access-Control-Allow-Credentials": true,
+        "x-api-key": apiKey,
+      },
+    };
+  fetch(
+  proxyurl +
+      endpoint +
+      `/applications/application?`+ // questions remain about this endpoint. These new endpoints currently cannot be found in AWS?!
```

Comment:
I've been slightly confused about this part. I couldn't find any of the endpoints that Dennis in the backend repo in our AWS. The API Service in AWS had an error for the PUT endpoints 
"The API with ID vewoyuxe51 doesn’t include a resource with path /accept having an integration arn:aws:lambda:us-east-2:885458936997:function:acceptApplications on the PUT method."

Reply:
SOLVED: endpoints in AWS created and connected to the lambda functions